manual/moderncv_userguide.tex

```diff
@@ -793,6 +865,36 @@ \subsubsection{Lengths in the skill matrix}
 \end{lstlisting}
 \code{<width>} and \code{<exp_width>} should be lengths smaller than \code{\\textwidth}, and \code{<factor>} must be between 0 and 1.
 
+\subsection{Additional link commands}%% adapted from Cristina Sambo's documentation
+To create textlinks to a website or a email address, use the following commands:
+A general weblink with an optional text.  
+\begin{lstlisting}
+  \link[<text>]{<link>}
+  %% example
+  \link[name of the link]{ftp://ftp.somesite.org}
+\end{lstlisting}
+
+A http link. The http prefix is generated automatically and is not needed in the link argument.
+\begin{lstlisting}
+  \link[<text>]{<link>}
+  %% example
+  \link[goto http side]{www.somehttpsite.org}
+\end{lstlisting}
+
+A https link. The https prefix is generated automatically and is not needed in the link argument.
+\begin{lstlisting}
+  \link[<text>]{<link>}
+  %% example
+  \httpslink[goto https site]{www.somesecuresite.org}
+\end{lstlisting}
+
+A e-mail link. The mailto prefix is generated automatically and is not needed in the link argument.
```

Comment:
```suggestion
An email link. The mailto prefix is generated automatically and is not needed in the link argument.
```
"email" rather than "e-mail" is consistently used throughout the rest of the doc